client/app/cloud/project/compute/infrastructure/privateNetwork/addVRack/cloud-project-compute-infrastructure-privateNetwork-addVRack.html

```diff
@@ -0,0 +1,60 @@
+            <div class="radio activate-private-network-radio">
+                <label class="full-width bottom-space-m8" data-ovh-stop-event="click" for="existingVrack">
+                    <input name="activateOptions" id="existingVrack"
+                        class="no-box-shadow vertical-text-bottom"
+                        data-ovh-stop-event="click"
+                        type="radio"
+                        data-ng-value="AddVRackCtrl.activateOptions.EXISTING"
+                        data-ng-model="AddVRackCtrl.selectedActivateOption">
+                    {{ :: "private_network_activate_existing" | translate }}
+                    <div class="activate-private-network-vRack-list">
+                        <div class="oui-select activate-private-network-select">
+                            <select class="oui-select__input"
+                                id="vRacks" name="vRacks"
+                                data-ng-model="AddVRackCtrl.selectedVrack"
+                                data-ng-options="vRack.displayName for vRack in AddVRackCtrl.vRacks"
+                                required />
+                            <i class="oui-icon oui-icon-chevron-down"></i>
+                        </div>
+                    </div>
+                </label>
+            </div>
+
+            <div class="radio activate-private-network-radio">
+                <label class="full-width bottom-space-m8" data-ovh-stop-event="click" for="newVrack">
+                    <input name="activateOptions" id="newVrack"
+                        class="no-box-shadow vertical-text-bottom"
+                        data-ovh-stop-event="click"
+                        type="radio"
+                        data-ng-value="AddVRackCtrl.activateOptions.NEW"
+                        data-ng-model="AddVRackCtrl.selectedActivateOption">
+                    {{ :: "private_network_activate_new" | translate }}
+                    <div class="activate-private-network-new-vRack-description">
+                        <p>{{ :: "private_network_activate_new_vRack_description" | translate }}</p>
+                    </div>
+                </label>
+            </div>
+        </form>
+    </div>
+    <div class="oui-modal__footer">
+        <button type="reset" class="oui-button oui-button_secondary"
+            data-ng-click="AddVRackCtrl.cancel()"
+            data-ng-disabled="AddVRackCtrl.loaders.activate">
+            <span data-translate="common_cancel"></span>
+        </button>
+        <button type="submit" class="oui-button oui-button_primary"
+            data-ng-click="AddVRackCtrl.activate()"
+            data-ng-disabled="AddVRackCtrl.loaders.activate">
+            <span data-translate="private_network_activate"></span>
+            <span data-ng-class="AddVRackCtrl.loaders.activate ? 'fa fa-circle-o-notch fa-spin' : ''"></span>
```

Comment:
There shoudln't be a spinner in the button.  Just disabled it or hide everything with http://master.ui-kit.ovh/#!/ovh-ui-kit/spinner

or go on https://www.ovh.com/manager/cloud/#/network/iplb/ip-217.182.130.226/home for an example.  Ciphers => click on action button => click edit and edit the cipher.. you'll see what I mean

Reply:
done.